site/docs/v1/tech/themes/template-variables.adoc

```diff
@@ -36,7 +36,7 @@ The application resolved by the provided [field]#client_id# provided on the requ
 The OAuth v2.0 `client_id` parameter. This is synonymous with FusionAuth's Application Id.
 
 [field]#currentUser# [type]#[User]# [since]#Available since 1.30.0#::
-When there is an active SSO session, this variable will contain the currently logged in user. When an SSO session does not yet exist, this variable will be undefined.
+When there is an active SSO session, this variable will contain the currently logged in user. When an SSO session does not yet exist, this variable will be null. If the user has not checked the `Keep me signed in` option, there is no SSO session and this variable will be null.
```

Comment:
null should be `null`.

Reply:
Updated and merged in a separate PR. Thanks!